faststream/_compat.py

```diff
@@ -79,6 +79,7 @@ def json_dumps(*a: Any, **kw: Any) -> bytes:
 
     major, minor, *_ = map(int, FASTAPI_VERSION.split("."))
     FASTAPI_V2 = major > 0 or minor > 100
+    FASTAPI_V106 = major > 0 or minor > 106
```

Comment:
Should be >= 106

Reply:
It also compatible with 1.* in the future

Reply:
Got it, done